src/pages/ProfileOrders/index.tsx

```diff
@@ -34,15 +78,22 @@ function ProfileOrders() {
         </TabButton>
       </TabsWrapper>
       <Grid container spacing={3}>
-        <Grid item>
-          <OrderCard />
-        </Grid>
-        <Grid item>
-          <OrderCard />
-        </Grid>
-        <Grid item>
-          <OrderCard />
-        </Grid>
+        {orders.length === noOrders ? (
+          <Grid item>
+            <Typography variant="h3">{t('profileOrders.noOrders')}</Typography>
```

Comment:
I asssume yo show it on each page, or at least you should do such. make a wrapper component, its not good practice put condition in each component

Reply:
But such orders are used only in this component in one place? Or do you mean to create a reusable grid wrapper that accepts data that should be rendered and the text that should be displayed if there is no data?

Reply:
yes, but message could be a props, in other component it could be - "no data in cart", or "no items in favorite" so on
I don't belive that in full project this is the only place where similar condition exist